src/project/internal/projectactionscontroller.cpp

```diff
@@ -1577,6 +1581,29 @@ void ProjectActionsController::warnScoreCouldnotBeSaved(const std::string& error
     interactive()->warning(muse::trc("project/save", "Your score could not be saved"), errorText);
 }
 
+void ProjectActionsController::warnScoreCorruptAfterSave(const Ret& ret)
+{
+    std::string errMessage = ret.toString();
+
+    std::any corruptedAfterMove = ret.data("corruptedAfterMove");
+    std::string originalFileCorruptedMessage = corruptedAfterMove.has_value() && std::any_cast<bool>(corruptedAfterMove)
+                                               ? "Unfortunately your original file has become corrupt.\n\n"
+                                               : "";
+
+    std::string message = muse::qtrc("project/save",
+                                      "An error occurred while saving your file. "
+                                      "This may be a one-off error. Please retry saving the file in order not to lose your latest changes.\n\n"
+                                      "If it keeps happening, try saving the file to a different location or to MuseScore.com. "
+                                      " You could also try saving the file as an uncompressed folder or XML.\n\n"
```

Comment:
...and leading space here

Reply:
I'm not sure that the uncompressed folder should be mentioned here. It's more of a dev tool, and there are a lot of issues around it too

Reply:
@RomanPudashkin What issues are there around it?

Reply:
Yeah I'd like to know because I've been using a lot (there are various adjustments I've been making to my scores by editing the mscx file!)

Reply:
Here is a list of them (incomplete, I suspect) - https://github.com/musescore/MuseScore/pull/18645. Right now, this feature is pretty clunky to use and in most cases, users won't even be able to figure out how to use it. What would be significantly better is to be able to save individual mscx files as suggested here - https://github.com/musescore/MuseScore/issues/15995, but that's a completely different story